blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -135,12 +141,6 @@ class LeadArt extends Component {
 						customFields={{
 							playthrough: !!customFields?.playthrough,
 						}}
-						displayTitle={!hideTitle}
```

Comment:
Why are these being removed?

Reply:
This was an accident mid-wip. And it's been fixed for the files affected